Homework-Functions/src/sciit/homework/functions/ex1/CalculateMax.java

```diff
@@ -0,0 +1,17 @@
+package sciit.homework.functions.ex1;
+
+public class CalculateMax {
+    public static void main(String[] args) {
+
+    }
+
+    public static double getMax(double nr1, double nr2) {
```

Comment:
Desi functioneaza si asa, metoda `getMax` ar fi trebuit scrisa in clasa `MaxOf3Numbers`, nu vad nici un motiv pentru care functia asta sa merite propriul fisier.